umake/ui/cli/__init__.py

```diff
@@ -157,6 +158,65 @@ def mangle_args_for_default_framework(args):
     return result_args
 
 
+def get_frameworks_list(args):
+    """
+    Get a frameworks list based on the arguments.
+    Multiple forms of the frameworks list can ge given:
+        - List with all frameworks
+        - List with just only installed frameworks
+        - List with just installable frameworks
+
+    Args:
+        args: It's the return of parse_args function. Just args.list, args.list_available and args.list_installed
+        are need to this function.
+
+    Return:
+        Give the frameworks list as a string
+    """
+    categories = list_frameworks()
+    print_result = str()
+
+    if args.list or args.list_available:
+        # Sort the categories to prevent a random list at each new program execution
+        for category in sorted(categories, key=lambda cat: cat["category_name"]):
+            if category["is_installed"] == 0:
```

Comment:
You are right about the example of "ide" category. If you want I can remove it from the display. maybe it would have be nice to have an other option to have a detailed display versus a compact display. But I think for now, there are already enough arguments. Maybe in a future, with the next major release like that:
`umake list --detailled`
`umake list-available --detailled`
`...`